### PR TITLE
Add UI support in admin UI for organization add, edit, delete scenarios

### PR DESCRIPTION
**Pull request by dulithsenanayake**
This PR adds UI support in admin UI to add, edit & delete organizations.

Fixes https://github.com/wso2/apk/issues/1144

### CODE REVIEW

**Review by Krishanx92 — APPROVED** [review]

File: admin/admin-ui/client/source/layout/MainLayout/Header/HeaderContent/Profile/index.tsx [hunk 1]
```diff
@@ -128,11 +128,11 @@ const Profile = () => {
                                             <Grid container justifyContent="space-between" alignItems="center">
                                                 <Grid item>
                                                     <Stack direction="row" spacing={1.25} alignItems="center">
-                                                        <Avatar alt="profile user" src='/images/avatar-1.png' sx={{ width: 32, height: 32 }} />
+                                                        <Avatar alt="profile user" sx={{ width: 32, height: 32 }} />
                                                         <Stack>
                                                             <Typography variant="h6">{user.user}</Typography>
                                                             <Typography variant="body2" color="textSecondary">
-                                                                UI/UX Designer
+                                                                Org1
```
Comment (RakhithaRR) [inline admin/admin-ui/client/source/layout/MainLayout/Header/HeaderContent/Profile/index.tsx:135]: Is this a hardcoded value?

File: admin/admin-ui/client/source/pages/Organizations/AddEditOrganization.tsx [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+                        disabled={saving}
+                    >
+                        {saving ? (<CircularProgress size={16} />) : (<>{'Save'}</>)}
+                    </Button>
+                </DialogActions>
+            </Dialog>
+        </>
+    );
+}
+export default AddEditOrganization;
```
Comment (RakhithaRR) [inline admin/admin-ui/client/source/pages/Organizations/AddEditOrganization.tsx:280]: Missing EOF newlines. Check other places also.

File: admin/admin-ui/client/source/components/data-table/PaginatedClientSide/SearchTable.tsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+ */
+
+import React, { useState } from 'react';
+import TextField from '@mui/material/TextField';
+import Autocomplete from '@mui/material/Autocomplete';
+import AppBar from '@mui/material/AppBar';
+import Toolbar from '@mui/material/Toolbar';
+import Grid from '@mui/material/Grid';
+import SearchIcon from '@mui/icons-material/Search';
+
```
Comment (chanaka3d) [inline admin/admin-ui/client/source/components/data-table/PaginatedClientSide/SearchTable.tsx:26]: Define a type for the props.

```
interface SearchTableProps {
  filter: string;
  setFilter: (filter: string) => void;
  searchProps: {
    searchPlaceholder: string;
  };
}
```

File: admin/admin-ui/client/source/components/data-table/PaginatedClientSide/SearchTable.tsx [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+export default function SearchTable({filter, setFilter, searchProps: {searchPlaceholder} }) {
+
```
Comment (chanaka3d) [inline admin/admin-ui/client/source/components/data-table/PaginatedClientSide/SearchTable.tsx:28]: Extract the onChange inline function and add type.

```javascript
const handleFilterChange = (e: ChangeEvent<HTMLInputElement>) => {
    setFilter(e.target.value);
  };
  ```

File: admin/admin-ui/client/source/layout/MainLayout/Header/HeaderContent/Profile/index.tsx (hunk 1 above)
Comment (dulithsenanayake) [reply admin/admin-ui/client/source/layout/MainLayout/Header/HeaderContent/Profile/index.tsx]: Here we should add the relevant organization. For that, there is a to-do implementation from the rest API side. So added a comment with this 20fa35609936424d278da31ea6c0b3d60d88b5ed

File: admin/admin-ui/client/source/pages/Organizations/AddEditOrganization.tsx (hunk 2 above)
Comment (dulithsenanayake) [reply admin/admin-ui/client/source/pages/Organizations/AddEditOrganization.tsx]: Fixed with 8ed4f174914dda0c6974e01d2deecce09f46359f

File: admin/admin-ui/client/source/components/data-table/PaginatedClientSide/SearchTable.tsx (hunk 3 above)
Comment (dulithsenanayake) [reply admin/admin-ui/client/source/components/data-table/PaginatedClientSide/SearchTable.tsx:25]: Fixed with 95fcc2e67312635ff5c352ad8e87c759b712bbd0

File: admin/admin-ui/client/source/components/data-table/PaginatedClientSide/SearchTable.tsx (hunk 4 above)
Comment (dulithsenanayake) [reply admin/admin-ui/client/source/components/data-table/PaginatedClientSide/SearchTable.tsx:39]: Fixed with 95fcc2e67312635ff5c352ad8e87c759b712bbd0

**Review by chanaka3d — APPROVED** [review]

**Pull request merged**